src/Appwrite/Platform/Modules/Proxy/Http/Rules/Verification/Update.php

```
@@ -74,6 +72,7 @@ public function action(
         Event $queueForEvents,
         Document $project,
         Database $dbForPlatform,
+        array $platform,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$platform'. (undefined)

(UnusedFormalParameter)